package.json

```diff
@@ -5,6 +5,8 @@
   "description": "A collection of gulp tasks.",
   "main": "index.js",
   "dependencies": {
+    "@frctl/fractal": "^1.1.7",
+    "@frctl/twig": "git+ssh://git@github.com:netzstrategen/twig-drupal.git",
```

Comment:
Isn't Fractal a project-specific dependency?

I understood this repo to be a generic gulp helper, not tied to any project.  Did I interpret that wrongly?

Reply:
Good point, but by now we don't have any condition in the fractal task  of this module to check if `@frctl/fractal` exists or not. So this would need some additional work and research on how to do this properly.

As this is the fastest, and simplest solution I suggest to proceed with this.

Reply:
Okay.  But in that case, I would remove the version restriction here and specify `"*"` instead, leaving the exact version requirement to the actual theme project?

Reply:
If using `*` there may be a version installed which includes breaking changes for the fractal task itself. E.g. rename functions, arguments etc. So this module will not work on it's own. We should leave it as it is and address this at a later point as it's currently not urgent nor blocking if we're chosing this way.